charts/app-config/values-integration.yaml

```
@@ -1825,6 +1825,8 @@ govukApplications:
             secretKeyRef:
               name: publishing-api-postgres
               key: DATABASE_URL
+        - name: ENQUEUE_PUBLISH_INTENTS
+          value: true
```

Can't remember off the top of my head whether bool true gets coerced to string somewhere along the way or not, but clearer to be explicit anyway.
```suggestion
          value: "true"
```

---

everything comes through to the apps as string values anyway, and the app is [checking for a string](https://github.com/alphagov/publishing-api/pull/2566/files#diff-b1570bf3b35a592d5767f288657003c340052a04b4480a34528327fee4f24a3fR9)

but I'm happy to be explicit :+1: